packages/fxa-auth-server/lib/senders/emails/templates/passwordChangeRequired/index.mjml

```diff
@@ -30,7 +30,7 @@
       </span>
       <br />
       <span data-l10n-id="passwordChangeRequired-signoff-name">
-        The Firefox Accounts Team
+        The Firefox accounts team
```

Comment:
"The Firefox accounts Team" didn't seem right, and neither did capitalizing Accounts here for this one instance, so I lowercased "accounts" and "team"